src/models/geograph.py

```diff
@@ -189,6 +190,11 @@ def crs(self):
         """Return crs of dataframe."""
         return self.df.crs
 
+    @property
+    def bounds(self):
+        """Return bounds of entire graph"""
+        return self.df.total_bounds
```

Comment:
Suggestion, change this to `self.df.sindex.bounds` 
![image](https://user-images.githubusercontent.com/13484097/111698457-c3378180-882e-11eb-84bf-a19f3a2cee2b.png)